homework/tests/test_task03.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+from homework.Tasks02_05.task03 import find_maximum_and_minimum
+
+
+def test_find_maximum_and_minimum_in_file_with_nums():
+    """Testing that function find min and max in file"""
+    text_file = open("text_with_nums.txt", "w")
+    text_file.write("1,2,3,5,613")
```

Comment:
in lecture about testing I told that it's bad practise to create files inside tests. line 15 will not be executed if test fails (on line 12). the environment where you run tests may not let you create files there. You should either mock file here to get these numbers using Mock or create this file in project